tests/eas/load_tracking.py

```diff
@@ -537,6 +540,406 @@ def test_load_avg_behaviour(self, experiment, tasks):
         """
         return self._test_behaviour(experiment, tasks, 'load_avg')
 
+class _CPUMigrationBase(LisaTest):
+    """Base class for shared functionality of tasks migration tests"""
+    test_conf = {
+        'tools'    : [ 'rt-app' ],
+        'ftrace' : {
+            'events' : [
+                'sched_switch',
+                'sched_load_cfs_rq',
+            ],
+        },
+        'modules': ['cpufreq', 'cgroups'],
+    }
+
+    # Allowed error margin
+    allowed_util_margin = UTIL_SCALE * 0.02
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        super(_CPUMigrationBase, cls).runExperiments(*args, **kwargs)
+
+    @classmethod
+    def generate_json(cls, test_env, tasks, workload_name):
+        """
+        Generate a json file which describes the different tasks.
```

Comment:
I think we usually (dunno why) keep the first line right after the three `"""`... I think it's a documentation processor "requirement"

Reply:
Done